cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -231,44 +226,38 @@
         }
     } );
 
-    // Desktop size.
-    .respond-to-min( @bp-med-min {
-      overflow: hidden;
-
-      // Match height of Search button at desktop size.
-      // Used to make overflow cover one line only.
-      // 1px offset is to expand overflow area by 1px
-      // so that the focus outline is not cropped.
-      max-height: 47px;
-      position: relative;
-      left: -1px;
-      top: -1px;
-
-      &_trigger,
-      &_trigger:focus,
-       {
-        position: relative;
-        top: 1px;
-      }
-      &_content {
-        position: relative;
-        left: 1px;
-      }
-
-
-      &_trigger {
-          &[aria-expanded="true"] {
-              display: none;
-          }
-      }
-
-      &_content {
-          padding-right: @margin_half__em;
-      }
+    // Desktop and above
+    .respond-to-min( @bp-med-min, {
+        // Always show on desktop, even without JS
+        display: block;
+        // Center on the CTA divider
```

Comment:
Maybe `Center on the call to action (CTA) divider to right of search.` to avoid jargon in the comments.